frontend/src/pull-card/index.tsx

```diff
@@ -64,7 +64,7 @@ export const PullCard = memo(function PullCard({
   const cardRef = useRef<HTMLElement>(null);
   highlightOnChange(cardRef, [pull.received_at]);
 
-  const linesAvailable = pull.additions && pull.deletions;
+  const linesAvailable = (pull.additions != null) && (pull.deletions != null);
```

Comment:
Should we show +0 or -0?

Reply:
I could create two const, showAdditions / showDeletions and opt to exclude the +0 or -0

Reply:
I think I like the format being consistent and always showing both. But I'm only once voice :-)

Reply:
Cleaned the logic up a bit by just showing additions/deletions if they are present

https://github.com/iFixit/pulldasher/pull/370/commits/c8439b6855874229afd720f27ec54dbfd746613b

Reply:
> Since we know these are numbers, I'd go with this:

I think that these are defaulted in the DB to be null, and usually need a refresh via the Pull Card to get the correct values set

![Untitled](https://user-images.githubusercontent.com/95656772/228981116-dbe46378-b568-407b-8421-bd92ed7e4bca.png)

That's the reason I used the explicit null check, so it won't display the + / - until they are the actual values